docs/leader_election.md

```diff
@@ -0,0 +1,104 @@
+# Increasing LND reliablity by clustering
+
+Normally LND nodes use the embedded bbolt database to store all important state.
```

Comment:
*states

Reply:
Not sure about this one actually but sgtm.